src/modules/common/components/Page.js

```diff
@@ -46,15 +47,26 @@ type PropType = {
 }
 
 class Page extends React.Component<PropType, StateType> {
+  // noinspection JSDuplicatedDeclaration
   onMessage: (event: WebViewMessageEvent) => void
+  // noinspection JSDuplicatedDeclaration
```

Comment:
try to avoid these with arrow-functions.

Reply:
Done.